.travis.yml

```diff
@@ -150,9 +150,19 @@ branches:
   only: [master]
 
 # MANUALLY ADDED. Do not remove.
-after_success:
-  - ./tool/deploy.sh
-# END MANUAL EDITS.
+deploy:
+  skip_cleanup: true
+  on:
+    repo: dart-lang/angular
+    #branch: master
+    branch: i978_better_deploy
+    condition: -d "examples/hacker_news_pwa/build/web"
+  provider: firebase
+  token:
+    # Created by kevmoo
+    secure: "onLskcM902pl6N8gtTp/oqRQmtLt6xjnJx8WW8r0tIaYdiQyhHn3sm6rFFyEiYBuIjpNS1+zK3pkDZ2dT1ZIVXPeQ7Zyo1TkbwqRDdWIuy9Qcv8RjAnRvhC2jh41e5nFuLQ1cNJQV6JtpF+sXfc7vpFs5Iaq+X9M4E7Z/HqHQ8IDRXXTLMvZpshSHr+ReaLzNp5p9ShxNNGEoDERZ4+O5sd39Al7fEoczSQNsq9BObWyhmR04lKBGLf7ECasp/qbrWExkiRODS1Izwd1LfLtHeNUoTHfbRs3tSxB74eVp+BBwtkni5Zp+llY/DFZgxw2GWC7iVT0PbjAqzdb+DKgiYjdIVC8wwOMV4C3eeZbwVGiAAQtt+NgpLKKpslp/+Cj+F8sDgSWeUJmtxQhNtV/sJDD+Ai4ijOubjba0WuGmuDgNzVWMPrsKNL+e6sMATR1omAWfUKIaYjAo0vk2KFoQbrwpl8qRQLrchrzgTi6JEs000cnVJYimlmcSK8dJdd1wkw/wiOAoD9oR9/ApUQaC7p8GnqIjhwR+dI5efj/zteBZI3HL/EbVEarpRlx4GKcCzYmC7ANSCJtex/d6fArBnWSKvoQHcedLkAbqpB622ir+d+QeRRSbyxSYkVgKFDNje8j32wZ3zNYXPqzwEPFy/ecpDTkrY9ltbZwncnsrtc="
```

Comment:
This is what's documented: https://docs.travis-ci.com/user/deployment/firebase/#Generating-your-Firebase-token

Wonder if it's necessary if have the key in settings...

Reply:
Why can't we go with the `$FIREBASE_TOKEN` approach? Seems easier.